src/wp-includes/block-patterns/heading.php

```diff
@@ -0,0 +1,16 @@
+<?php
+/**
+ * Heading.
+ *
+ * @package WordPress
+ */
+
+return array(
+	'title'       => _x( 'Heading', 'Block pattern title', 'default' ),
```

Comment:
I think these 'default' domain arguments should be dropped since we're already on Core.

Reply:
Maybe it's fine though. @swissspidy @gziolo do you know what is the guideline for Core?

Reply:
It defaults to `default` so it can be dropped and in practice it usually is.

Reply:
Thanks for the heads up, I just removed all of them 👍